src/test/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtilsTest.java

```diff
@@ -86,27 +81,27 @@ public static void setupBeforeAll(@Autowired PredefinedShelfService predefinedSh
         setBooksInPredefinedShelves();
     }
 
-    private static void findPredefinedShelves(PredefinedShelfService predefinedShelfService) {
+    private void findPredefinedShelves(PredefinedShelfService predefinedShelfService) {
         toReadShelf = predefinedShelfService.findToReadShelf();
         readShelf =  predefinedShelfService.findReadShelf();
         didNotFinishShelf = predefinedShelfService.findDidNotFinishShelf();
     }
 
-    private static void resetBookRepository() {
+    private void resetBookRepository() {
         bookRepository.deleteAll();
     }
 
-    private static void setBooksInPredefinedShelves() {
+    private void setBooksInPredefinedShelves() {
         toReadShelf.setBooks(Set.of(book1, book2));
         readShelf.setBooks(Set.of(book3));
         didNotFinishShelf.setBooks(Set.of(book4));
     }
 
-    private static void createAndSaveBooks() {
-        book1 = bookRepository.save(new Book("someTitle", NO_AUTHOR, toReadShelf));
-        book2 = bookRepository.save(new Book("someTitle2", NO_AUTHOR, toReadShelf));
-        book3 = bookRepository.save(new Book("someOtherTitle", NO_AUTHOR, readShelf));
-        book4 = bookRepository.save(new Book("yetAnotherTitle", NO_AUTHOR, didNotFinishShelf));
+    private void createAndSaveBooks() {
+        book1 = bookRepository.save(new Book("someTitle", null, toReadShelf));
+        book2 = bookRepository.save(new Book("someTitle2", null, toReadShelf));
+        book3 = bookRepository.save(new Book("someOtherTitle", null, readShelf));
+        book4 = bookRepository.save(new Book("yetAnotherTitle", null, didNotFinishShelf));
     }
```

Comment:
Any reason for changing from more readable NO_AUTHOR to explicitly passing null?